instana/instrumentation/logging.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+
+import wrapt
+import logging
+import sys
+
+from ..log import logger
+from ..singletons import tracer
+
+@wrapt.patch_function_wrapper('logging', 'Logger._log')
+def log_with_instana(wrapped, instance, argv, kwargs):
+    # argv[0] = level
```

Comment:
We should wrap this in a try/except/finally to protect against anything going wrong and not losing the log message for the end user.